test/package.json

```diff
@@ -23,7 +23,7 @@
     "pretest": "npm run build",
     "test": "npm run test:regression && npm run test:visreg",
     "test:regression": "ava node built/regression/generateRegressionTests.js --verbose --serial",
-    "test:visreg": "ava built/visreg/{chrome,firefox}.js --verbose --serial",
+    "test:visreg": "ava built/visreg/firefox.js --verbose --serial",
```

Comment:
Please put this back.

Reply:
Thank you Andrew, reverted the required changes.